iXBRLViewerPlugin/viewer/src/js/inspector.js

```diff
@@ -529,7 +529,7 @@ Inspector.prototype._updateValue = function (item, showAll, context) {
     }
 
     var valueSpan = $('tr.value td .value', context).empty().text(v);
-    if (item instanceof Fact && item.isNil()) {
+    if (item instanceof Fact && (item.isNil() || item.isInvalidIXValue())) {
```

Comment:
Nit: Perhaps an `isValidFact` function?

Reply:
What would that do?  If you mean it to check nil also, then nil is a valid fact value, so that wouldn't help here.  If you mean it to check for other kinds of invalid state (e.g., invalid unit), it seems wrong to add `<i>` emphasis to the value here just because something else is invalid.

Reply:
Sorry, misunderstanding